spec/support/employee_box.cr

```diff
@@ -0,0 +1,26 @@
+class EmployeeBox
+  getter form
+
+  @form = Employee::BaseForm.new
```

Comment:
If you're up for it, maybe you could extract a `BaseBox` class? Something like this: https://gist.github.com/paulcsmith/c5da6e8888d1a50f13fb6d9acc37a6a9#file-base_box-cr